clients/java/src/main/java/io/zeebe/client/impl/oauth/OAuthCredentialsProvider.java

```diff
@@ -188,9 +187,7 @@ private ZeebeClientCredentials fetchCredentials() throws IOException {
 
     try (final InputStream in = connection.getInputStream();
         final InputStreamReader reader = new InputStreamReader(in, StandardCharsets.UTF_8)) {
-
-      final ZeebeClientCredentials fetchedCredentials =
-          CREDENTIALS_READER.readValue(CharStreams.toString(reader));
+      final ZeebeClientCredentials fetchedCredentials = CREDENTIALS_READER.readValue(reader);
```

Comment:
I'm not familiar with the credentials reader. Is this logically equivalent?

Reply:
Yes, it can also take an InputStreamReader instead of a string. Also the tests pass and this is relatively well-covered so I think there's no issue =)

Reply:
In that case consider this fully approved :)